README.md

```diff
@@ -534,15 +534,15 @@ end
 ## Development
 
 After checking out the repo, run `bin/setup` to install dependencies.
-Then, run `rake spec` to run the tests.
+Then, run `bundle exec rake spec` to run the tests.
```

Comment:
At least in my machine I needed to add exec and other places in this readme also uses exec so I think this is the safest